database-service/scripts/waitForService.sh

```diff
@@ -9,8 +9,8 @@ wait_for_port() {
     for i in `seq 1 $HASURA_GRAPHQL_MIGRATIONS_SERVER_TIMEOUT`;
     do
         echo "waiting for graphql engine"
-        nc -z $HOST $PORT
-        [ "$?" == "0" ] && return || sleep 1
+        version=$(curl -s http://$HOST:$PORT/v1/version | jq ".version")
+        [ "$version" == "" ] && sleep 1 || exit 0
```

Comment:
Previously, the tests failed mostly because this check was wrong. Now it should be fine.